File: Day_7/JS/TaskSix.js
let number = 1;

while( number <= 10 ){
	
	if(number % 4 == 0){
		let constant_number = number
		let count = 0;
		while (count < 5){
			console.log(number);
			number *= constant_number;
			count++;
			
		}
		number = constant_number;
	}
	number++;

}	
	
	

File: Day_7/Java/TaskSix.java
public class TaskSix{
	public static void main(String [] args){
		int parenthesis = 0;
		int constantNumber = 0;
		for(int number = 1; number <= 10; number++){
			if( number % 4 == 0 ){
				constantNumber = number;
				if (parenthesis < 1){
					parenthesis ++;
					System.out.print("(");}
			for(int count = 1; count <= 5; count++){
				System.out.printf("%d ", number);
				number *= constantNumber;}
			number = constantNumber;
			System.out.print(" ");		
			parenthesis++;
			if (parenthesis == 3 )
			System.out.print(")");}

		
		}	
		

	}

}